### PR TITLE
Implement testing for quiz and update QuizCommand class

### PR DESCRIPTION
**Pull request by tengjianling**
Add QuizCommandTest, QuizCommandParserTest, ExitQuizCommandTest, ExitQuizCommandParserTest.

### CODE REVIEW

File: src/test/java/seedu/medmoriser/logic/parser/QuizCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        assertParseFailure(parser, "     ", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                QuizCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_validArgsQAndAContainsKeywordsPredicate_returnsQuizCommand() {
+        // no leading and trailing whitespaces
+        QuizCommand expectedQuizCommand =
+                new QuizCommand(new QAndAContainsKeywordsPredicate(Arrays.asList("digestive", "skeletal")));
+        System.out.println(expectedQuizCommand);
```
Comment (jonfoocy) [inline src/test/java/seedu/medmoriser/logic/parser/QuizCommandParserTest.java:31]: Extra print statement?

File: src/test/java/seedu/medmoriser/logic/parser/QuizCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        // no leading and trailing whitespaces
+        QuizCommand expectedQuizCommand =
+                new QuizCommand(new TagContainsKeywordsPredicate(Arrays.asList("t", "digestive system",
+                        "skeletal system")));
+        assertParseSuccess(parser, "t/digestive system, skeletal system", expectedQuizCommand);
+
+        // multiple whitespaces between keywords
+        assertParseSuccess(parser, "t/ \n digestive system, \n \t skeletal system  \t", expectedQuizCommand);
+    }
+}
```
Comment (jonfoocy) [inline src/test/java/seedu/medmoriser/logic/parser/QuizCommandParserTest.java:61]: Can consider adding test cases for parse failures other than an empty argument, either in this PR or subsequently

**Review by jonfoocy — COMMENTED** [review]
Generally looks great, just minor issues

**Review by yongmingyang — APPROVED** [review]
LGTM, just minor changes pointed out by jonathan

**Review by jonfoocy — APPROVED** [review]
